package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@aserto/aserto-react-components",
-  "version": "7.2.3",
+  "version": "7.2.4",
```

Since we're adding new functionality, we should go with a minor version bump
```suggestion
  "version": "7.3.0",
```